<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diagramWidget</class>
 <widget class="QWidget" name="diagramWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Diagram viewer</string>
  </property>
  <widget class="QStackedWidget" name="diagramHolder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QToolButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>820</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Previous diagram</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="arrowType">
    <enum>Qt::LeftArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>820</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Next diagram</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="arrowType">
    <enum>Qt::RightArrow</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="showOriginalSeries">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>810</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Original series</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showSHCModels">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>830</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SHC models</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="currChartText">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>830</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Chart 1/1</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="chartInfo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>810</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showSI">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>850</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sigma-index</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
